src/config.rs

```diff
@@ -0,0 +1,34 @@
+use clap::Parser;
+
+/// Define a struct to hold all command-line arguments
+#[derive(Debug, Parser)]
+#[command(name = "Nomad External DNS Tool For Hetzner", version = "0.1.0", about = "Automatically updates DNS records", long_about = None)]
```

Comment:
You don't need to set these. They should be inferred from the `Cargo.toml`. See [here](https://github.com/svenstaro/wiresmith/blob/76cf7c3cef0ec9b7105fe865f04d99e92f7d365b/src/args.rs#L15).